docs/nodevertical.md

```diff
@@ -35,21 +35,33 @@ Default: `quay.io/openshift-scale/scale-ci-workload`
 Container image that runs the workload script.
 
 ### WORKLOAD_JOB_NODE_SELECTOR
-Default: `true`  
+Default: `false`  
 Enables/disables the node selector that places the workload job on the `workload` node.
 
 ### WORKLOAD_JOB_TAINT
-Default: `true`  
+Default: `false`  
 Enables/disables the toleration on the workload job to permit the `workload` taint.
 
 ### WORKLOAD_JOB_PRIVILEGED
-Default: `true`  
+Default: `false`  
 Enables/disables running the workload pod as privileged.
 
 ### KUBECONFIG_FILE
 Default: `~/.kube/config`  
 Location of kubeconfig on orchestration host.
 
+### PBENCH_INSTRUMENTATION
```

Comment:
The docs of other workloads needs to be modified with the new vars ( PBENCH_INSTRUMENTATION and ENABLE_PBENCH_COPY ). Is the plan to create a PR later?

Reply:
Eventually yes however those workloads do not implement these vars in the same fashion yet.  I am a bit conflicted on structuring the pbench_* vars as there are a few combinations we will want. (`PBENCH_INSTRUMENTATION`, `ENABLE_PBENCH_AGENTS`, and `ENABLE_PBENCH_COPY`)  For now lets keep this with the nodevertical workload and I will see about extending it to the podvert and mastervert workloads as well.  It is a little bit unclear how we can run with the network tests but I will explore this more once we land this.

The other thought is setting defaults to not require pbench and to be more or less for CI runs.  Thanks for reviewing this.

Reply:
Got it.